src/renderer/RecorderPreview.tsx

```diff
@@ -52,6 +52,7 @@ const RecorderPreview: React.FC = () => {
         border: '2px solid black',
         boxSizing: 'border-box',
       }}
+      className="z-0"
```

Comment:
Does this make any difference?

Reply:
Oh, no, left over from when I was trying to get this preview to be less problematic 😀